packages/blob-storage-manager/src/index.test.ts

```diff
@@ -0,0 +1,62 @@
+import { afterEach, describe, expect, it, vi } from "vitest";
+
+import { BlobStorageManager } from "./BlobStorageManager";
+import {
+  createOrLoadBlobStorageManager,
+  resetBlobStorageManager,
+} from "./index";
+import { GoogleStorage, PostgresStorage, SwarmStorage } from "./storages";
+
+vi.mock("./BlobStorageManager", async () => {
+  const manager = await vi.importActual<BlobStorageManager>(
+    "./BlobStorageManager"
+  );
+  return {
+    BlobStorageManager: vi.fn().mockImplementation(() => manager),
+  };
+});
+
+vi.mock("./storages", async () => {
+  // eslint-disable-next-line @typescript-eslint/consistent-type-imports
+  const storages = await vi.importActual<typeof import("./storages")>(
+    "./storages"
+  );
+  return {
+    GoogleStorage: {
+      tryCreateFromEnv: vi.fn().mockResolvedValue(storages.GoogleStorage),
+    },
+    SwarmStorage: {
+      tryCreateFromEnv: vi.fn().mockResolvedValue(storages.SwarmStorage),
+    },
+    PostgresStorage: {
+      tryCreateFromEnv: vi.fn().mockResolvedValue(storages.PostgresStorage),
+    },
+  };
+});
+
+describe("createOrLoadBlobStorageManager", () => {
```

Comment:
Let's keep this function untested for now.

These unit tests are too implementation specific so I'd just remove them